siriuspy/siriuspy/orbintlk/csdev.py

```diff
@@ -107,6 +107,7 @@ class Const(_csdev.Const):
         ('RTMFreqIntgGain-SP', 128),
         ('RTMPhaseNavg-SP', 0),
         ('RTMPhaseDiv-SP', 0),
+        ('UpstreamDebugEn-Sel', 0),
```

Comment:
I think there should be a comment here explaining our rationale for not locking the value for all crates: we don't want to kill the beam when a timing device outside of the critical path fails, and locking would trigger that, unless we further complicated the locking logic.

We should also update (at least) the orbintlk GUI button that says these PVs refer to the RTM loop.